src/Shopsys/ShopBundle/DataFixtures/Demo/UserDataFixture.php

```diff
@@ -11,17 +11,22 @@
 use Shopsys\FrameworkBundle\Component\DataFixture\AbstractReferenceFixture;
 use Shopsys\FrameworkBundle\Component\Domain\Domain;
 use Shopsys\FrameworkBundle\Component\String\HashGenerator;
+use Shopsys\FrameworkBundle\Model\Customer\CustomerDataFactory;
 use Shopsys\FrameworkBundle\Model\Customer\CustomerFacade;
 use Shopsys\FrameworkBundle\Model\Customer\CustomerPasswordFacade;
 use Shopsys\FrameworkBundle\Model\Customer\User;
 
 class UserDataFixture extends AbstractReferenceFixture implements DependentFixtureInterface
 {
     public const USER_WITH_RESET_PASSWORD_HASH = 'user_with_reset_password_hash';
+    public const USER_WITH_10_PERCENT_DISCOUNT = 'user_with_10_percent_discount';
 
     /** @var \Shopsys\FrameworkBundle\Model\Customer\CustomerFacade */
     protected $customerFacade;
 
+    /** @var \Shopsys\FrameworkBundle\Model\Customer\CustomerDataFactory */
```

Comment:
Is there some reason why the annotations are different from all other files?
```suggestion
    /**
     * @var \Shopsys\FrameworkBundle\Model\Customer\CustomerDataFactory
     */
```

Reply:
Yes, they are usually inlined because they are not in class definition (eg. above properties) but in the code blocks itself. It's consistent across the codebase.